demos/projects/ESPRESSIF/aziotkit/README.md

```diff
@@ -14,6 +14,8 @@ IoT Hub | DPS
 Have an [Azure IoT Hub](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal) created | Have an instance of [IoT Hub Device Provisioning Service](https://docs.microsoft.com/azure/iot-dps/quick-setup-auto-provision#create-a-new-iot-hub-device-provisioning-service)
 Have a [logical device](https://docs.microsoft.com/azure/iot-hub/iot-hub-create-through-portal#register-a-new-device-in-the-iot-hub) created in your Azure IoT Hub using your preferred authentication method | Have an [individual enrollment](https://docs.microsoft.com/azure/iot-dps/how-to-manage-enrollments#create-a-device-enrollment) created in your instance of DPS using your preferred authentication method
 
+**Instructions on how to create an X.509 cert for tests can be found [here](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md#configure-and-run-the-samples) (Step 1). Please note that you might need to install some of the [prerequisites](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md#prerequisites) like OpenSSL.** 
+
 *While this sample supports SAS keys and Certificates, for simplification, this guide will refer only to SAS keys.

```

Comment:
```suggestion

```
This had been replaced in main with the above block before my updates to styling here went in, so I think it should probably stay removed

Reply:
Sorry, should have phrased this more clearly! The `While this sample supports SAS keys and Certificates, for simplification, this guide will refer only to SAS keys.` should be removed - the instructions for the x.509 cert should stay